frontend/packages/console-shared/src/constants/common.ts

```diff
@@ -78,3 +78,5 @@ export enum FLAGS {
   CONSOLE_EXTERNAL_LOG_LINK = 'CONSOLE_EXTERNAL_LOG_LINK',
   CONSOLE_YAML_SAMPLE = 'CONSOLE_YAML_SAMPLE',
 }
+
+export const CONFIG_STORAGE_PERSPECTIVE_CONSOLE = 'console';
```

Comment:
Why is this named `CONFIG_STORAGE_PERSPECTIVE_CONSOLE` with `PERSPECTIVE` in it?